pkg/resolution/resolver/git/resolver.go

```diff
@@ -30,45 +31,73 @@ import (
 	gitcfg "github.com/go-git/go-git/v5/config"
 	"github.com/go-git/go-git/v5/plumbing"
 	"github.com/go-git/go-git/v5/storage/memory"
+	"github.com/jenkins-x/go-scm/scm/factory"
```

Comment:
curious why you're using jenkins-x/go-scm instead of https://github.com/drone/go-scm (the one linked to in the docs)?

Reply:
I link to the drone one? Lemme fix that, should be jenkins-x. That's the one I _know_ works, since I spent months working on it. =)